### DISCUSSION TITLE
Add back CockroachDB in the docs

### DISCUSSION THREAD

**Discussion by Yberion** [zitadel/zitadel#7425] [Ideas]
Before https://github.com/zitadel/zitadel/pull/7358, the docs was using CockroachDB as example.

Now it seems that it is using `PostgreSQL` by default, example https://zitadel.com/docs/self-hosting/deploy/compose.

This does not really fit with the recommandation https://zitadel.com/docs/self-hosting/manage/production#prefer-cockroachdb.

If this is recommanded to use `CockroachDB` in production, I should be able to setup a local env with the same "setup".

**Is it possible to provide both `CockroachDB` and `PostgreSQL` setup in the documentation ?**

This way, people that already started using a local `CockroachDB` setup can still create new instances with the same setup, updated by the Zitadel team.

I myself started to use the Docker compose that contained the CockroachDB setup, now I'm a bit confused because the doc defaults to PostgreSQL without the CockroachDB alternative.

**Comment by fforootd** [comment]
Thanks for bringing this up. We are really just in the middle of changing our documentation and this slipped somehow 😓 

So let me tell you how we see this 😁 

From now on forward we recommend PostgreSQL as default storage layer and we will update our documentation to reflect this.
We recommend CockroachDB in cases where a customer is interested in higher availability guarantees at the cost of query latency.

In other words, if you want to have lower latency -> PostgreSQL, if you want to have higher availability -> CockroachDB

On the subject of having both:
> Is it possible to provide both CockroachDB and PostgreSQL setup in the documentation ?

Yes we try to address this in a more clear way.

Hope this make things clearer, Flo

**Reply by Yberion** [reply]
Hello @fforootd,

That's awesome, thanks for the answer. 

It's clearer now, I'll change my local setup to use PostgreSQL 😃.

By the way, good job for the documention!

**Reply by fforootd** [reply]
Thank you, please feel free to share if you have more inputs 😁
